main.py

```diff
@@ -146,20 +148,24 @@ def main(arg_list: list[str] | None = None) -> None:
 
   """
 
-  args = parse_args(arg_list)
+  try:
+    args = parse_args(arg_list)
 
-  config = build_abcd_params_config(args)
+    config = build_abcd_params_config(args)
 
-  start_time = time.time()
-  print("Starting ABCD assessment... \n")
+    start_time = time.time()
+    logging.info("Starting ABCD assessment... \n")
 
-  if config.video_uris:
-    execute_abcd_assessment_for_videos(config)
-    print("Finished ABCD assessment. \n")
-  else:
-    print("There are no videos to process. \n")
+    if config.video_uris:
+      execute_abcd_assessment_for_videos(config)
+      logging.info("Finished ABCD assessment. \n")
+    else:
+      logging.info("There are no videos to process. \n")
 
-  print(f"ABCD assessment took - {(time.time() - start_time) / 60} mins. - \n")
+    logging.info("ABCD assessment took - %s mins. - \n", (time.time() - start_time) / 60)
+  except Exception as ex:
+    logging.info("ERROR: %s", ex)
```

Comment:
It may be "logging.error" instead of "logging.info" within Exception.

Reply:
done!